tests/source/code.js

```diff
@@ -48,6 +48,18 @@ class ContainingClass {
     }
 
     /**
+     * Another.
+     */
+    anotherMethod() {
+    }
```

Comment:
I wanted a couple more methods in this class, to make my test for the feature convincing enough. I updated the tests that this affected as well.

Reply:
Yeah, I'm not proud of this shared test corpus. :-(